modules/gatsby/website-template/src/components/site-query.jsx

```diff
@@ -0,0 +1,102 @@
+// This component contains the StaticQuery needed to provide data for the layout components.
```

Comment:
We might want to have ocular init copy this one from the source tree rather than duplicate it into templates?